### ISSUE TITLE
Cannot remember window position and size

### ISSUE THREAD

**Issue by tkp206093** [alexkay/spek#296]
Using Win11 23H2 3840X2160 150%

every time open spek is so small and In the upper left corner

**Comment by Safari77** [comment]
I have added a feature to write widthXheight into config file
https://github.com/Safari77/spek
However, something in spek reduces  width by 90 pixels, and height by 99 and then 138 pixels after starting. If someone knows how to fix that, it would be nice.

**Comment by kedaitinh12** [comment]
Try this version
https://github.com/MikeWang000000/spek-X

**Comment by Safari77** [comment]
> Try this version https://github.com/MikeWang000000/spek-X

Is there a commit which fixes this bug?

**Comment by kedaitinh12** [comment]
> > Try this version https://github.com/MikeWang000000/spek-X
> 
> Is there a commit which fixes this bug?

I think not but this app still update in the future, can you add your commits to this version?

**Comment by Safari77** [comment]
> > > Try this version https://github.com/MikeWang000000/spek-X
> > 
> > 
> > Is there a commit which fixes this bug?
> 
> I think not but this app still update in the future, can you add your commits to this version?

No, I can't, I do not have write access to the repo, MikeWang000000 does.
